web/src/main/java/org/springframework/security/web/servletapi/SecurityContextHolderAwareRequestFilter.java

```diff
@@ -27,12 +27,14 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.springframework.security.authentication.AuthenticationDetailsSource;
```

Comment:
Will you please add a test to `SecurityContextHolderAwareRequestFilterTests` that demonstrates the `AuthenticationDetailsSource` is being used?